uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/PlaceManagerImpl.java

```diff
@@ -563,6 +597,20 @@ public boolean closeAllPlacesOrNothing() {
         return true;
     }
 
+    @Override
+    public List<PlaceRequest> getUncloseablePlaces() {
```

Comment:
@paulovmr could you please add docs of the usage of this?

Reply:
Added in the `PlaceManager` interface.